packages/cf-component-label/package.json

```diff
@@ -10,6 +10,10 @@
   "publishConfig": {
     "registry": "http://registry.npmjs.org/"
   },
+  "dependencies": {
+    "cf-style-container": "^1.0.0",
```

Comment:
You don't need to change these manually. Lerna would handle this during the release.